config/api/src/main/java/org/jboss/windup/config/AbstractPathConfigurationOption.java

```diff
@@ -65,12 +66,14 @@ else if (!Files.exists(path))
         return ValidationResult.SUCCESS;
     }
 
+    @SuppressWarnings("rawtypes")
     @Override
     public ValidationResult validate(Object fileObject)
     {
-        if (fileObject == null && isRequired())
+        if ( (fileObject == null && isRequired()) ||
+             (fileObject instanceof Collection && isRequired() && ((Collection) fileObject).isEmpty()) )
         {
-            return new ValidationResult(ValidationResult.Level.ERROR, getName() + " is required!");
+            return new ValidationResult(ValidationResult.Level.ERROR, getLabel() + " must be specified.");
         }
```

Comment:
I'm not sure about this. Isn't name what's used for the actual user input on command line?

Reply:
Well, there are mixed/not consistent validation messages. Somewhere it is used just strings not related to name/label.

Reply:
Then probably we should make it consistent? Changing this will fix the inconsistency?

Reply:
+1 - It should always display the parameter name, IMO.

Reply:
ok, then the mixed validation message in https://github.com/windup/windup/blob/master/exec/api/src/main/java/org/jboss/windup/exec/configuration/options/OutputPathOption.java#L132 and above about `--output` are little different too

Reply:
I think some of those should display the parameter as well. Some of them are about logical errors, though, and the parameter is a little less important (since the user has clearly specified both already).

Reply:
if we are fine with such inconsistencies I will change it back.